@navikt/core/css/darkside/form/select.darkside.css

```diff
@@ -1,115 +1,113 @@
 .navds-select__input {
   appearance: none;
-  background-color: var(--ac-select-bg, var(--a-surface-default));
-  border-radius: var(--a-border-radius-medium);
-  border: 1px solid var(--ac-select-border, var(--a-border-default));
-  color: var(--ac-select-text, var(--a-text-default));
+  background-color: var(--ax-bg-input);
+  border-radius: var(--ax-border-radius-medium);
```

Comment:
Is this the correct border-radius?

Reply:
Yes, the larger radius is currently just experimental in Figma
<img width="367" alt="Screenshot 2024-11-25 at 16 45 36" src="https://github.com/user-attachments/assets/9c0ff680-5038-4ad0-bdb8-b0d6a6154759">